moveit_core/robot_state/src/robot_state.cpp

```diff
@@ -1357,91 +1357,78 @@ bool RobotState::getJacobian(const JointModelGroup* group, const LinkModel* link
     return false;
   }
 
-  if (!group->isLinkUpdated(link->getName()))
+  const std::set<const moveit::core::LinkModel*>& descendant_links = group->getUpdatedLinkModelsSet();
```

Comment:
Is it worth creating an alias for `std::set<const moveit::core::LinkModel*>` for readability?

Reply:
created a small local alias for moveit::core, so it reads:
`std::set<const LinkModel*>`

I think it's good to see the explicit type here, otherwise could just use `auto`.